src/playlist/playlistmanager.cpp

```diff
@@ -237,6 +238,30 @@ void PlaylistManager::Save(const int id, const QString &filename, const Playlist
 
 }
 
+bool PlaylistManager::CreateAPlaylistsDirectory(const QString& playlist_folder_path) {
+  bool success = QDir(playlist_folder_path).mkpath(playlist_folder_path);
+  if (!success) {
+    qLog(Debug) << "Could not create folder: " << playlist_folder_path;
+  }
+  return success;
+
+}
+
+void PlaylistManager::SaveAllPlaylists() {
+
+  QString playlist_folder_path = QFileDialog::getExistingDirectory(nullptr, tr("Open Directory"),QDir::home().path(),QFileDialog::ShowDirsOnly| QFileDialog::DontResolveSymlinks);
+  if (!CreateAPlaylistsDirectory(playlist_folder_path)) {
```

Comment:
`QFileDialog::getExistingDirectory` returns an existing directory, so this should be unnecessary.